pkg/module/go_get_fetcher.go

```diff
@@ -120,6 +112,25 @@ func getSources(goBinaryName string, fs afero.Fs, gopath, repoRoot, module, vers
 	return nil
 }
 
+func prepareEnv(gopath string) []string {
+	pathEnv := fmt.Sprintf("PATH=%s", os.Getenv("PATH"))
+	gopathEnv := fmt.Sprintf("GOPATH=%s", gopath)
+	cacheEnv := fmt.Sprintf("GOCACHE=%s", filepath.Join(gopath, "cache"))
+	disableCgo := "CGO_ENABLED=0"
+	enableGoModules := "GO111MODULE=on"
+	cmdEnv := []string{pathEnv, gopathEnv, cacheEnv, disableCgo, enableGoModules}
+	// add Windows specific ENV VARS
+	// if this one is missing we will get an error from go
+	if val, ok := os.LookupEnv("USERPROFILE"); ok {
```

Comment:
Is it a good idea to say to panic on windows if the env vars do not exist?

Reply:
For that one yes, or in some other way shutdown as it won't work without it.

Reply:
But should we do this here? Maybe some check at the top of stack?

Reply:
yep! I might have something in init() of proxy that determine if we have sane defaults to run the app?

Reply:
Maybe a different PR?